src/FacebookAds/Object/AbstractObject.php

```diff
@@ -45,6 +45,8 @@ public function __construct() {
   public function __get($name) {
     if (array_key_exists($name, $this->data)) {
       return $this->data[$name];
+    } elseif (in_array($name, static::getFieldsEnum()->getValues())) {
```

Comment:
I think i found  the issue, which is not actually here. but in AbstractObject::setData() that will override the $this->data structure.
https://github.com/facebook/facebook-php-ads-sdk/commit/134eca31ad46821f7d12efc743a87b4900296de9 should fix it. would you mind try it?